### DISCUSSION TITLE
Not Authorizing after deployed

### DISCUSSION THREAD

**Discussion by macooper59** [wg-easy/wg-easy#1461] [Q&A] [answered]
Hello Everyone,

I used the docker-compose to deploy this and I need to password protect it so I used the bcrypt script to hash my password. After that I deployed the container and got this when I went to the url: 
![image](https://github.com/user-attachments/assets/e2a1bbc6-9dcf-4f6b-a520-87ea9613d544)

Does anyone have any ideas? The following is the docker-compose:
`volumes:
  etc_wireguard:

services:
  wg-easy:
    environment:
      # Change Language:
      # (Supports: en, ua, ru, tr, no, pl, fr, de, ca, es, ko, vi, nl, is, pt, chs, cht, it, th, hi, ja, si)
      - LANG=en
      # ⚠️ Required:
      # Change this to your host's public address
      - WG_HOST=wg01.pihobby.loc

      # Optional:
      # - PASSWORD=D0v@N3w2d@y
      - PASSWORD_HASH='$2a$12$hb1voIYfYWimo4SH769RYuAsW6NOOkeYPPdo8ddRxYT28ycehtGIW'
      # - PORT=51821
      # - WG_PORT=51820
      # - WG_CONFIG_PORT=92820
      # - WG_DEFAULT_ADDRESS=10.8.0.x
      # - WG_DEFAULT_DNS=1.1.1.1
      # - WG_MTU=1420`

**Comment by shalafi99** [comment, marked as answer]
From [How_to_generate_an_bcrypt_hash.md](https://github.com/wg-easy/wg-easy/blob/master/How_to_generate_an_bcrypt_hash.md):

_Important : Please note: don't wrap the generated hash password in single quotes when you use docker-compose.yml. Instead, replace each $ symbol with two $$ symbols. For example:

- PASSWORD_HASH=$$2y$$10$$hBCoykrB95WSzuV4fafBzOHWKu9sbyVa34GJr8VV5R/pIelfEMYyG_

**Comment by macooper59** [comment]
Hello,

Okay thank you very much I will do that. I appreciate your help.

Michael Cooper
________________________________
